### DISCUSSION TITLE
Assenze non concorrono al calcolo

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#145] [Idee e proposte]
Buongiorno,sarebbe possibile implementare una funzione che ad esempio non consideri l'assenza concorrente nel calcolo ?Parlo ad esempio quando si è impegnati nei progetti scolastici mattutini e lo studente risulta assente , ma dal momento che si tratta sempre di un'attività scolastica...

**Comment by trinko** [comment]
E' la gestione del "fuori classe": alunno assente in classe ma presente in attività altrove.

Sicuramente sarà inserita, ne abbiamo bisogno anche noi, ma non ho idea dei tempi.
